recipe-page.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE>
+<html>
+  <head>
+    <link href='http://fonts.googleapis.com/css?family=Abel|Yellowtail|Droid+Serif' rel='stylesheet' type='text/css'>
+    <title>Delicious Recipes</title>
+    <link href="style.css" type="text/css" rel="stylesheet" />
+  </head>
+  <div class='container'>
+    <body>
+      <header>
+        <img src='challenge/logo.png' />
```

Comment:
The ending "/" is not longer necessary at the end of self closing tags (such as images)